.github/labeler-config.yml

```
@@ -0,0 +1,5 @@
+# Add 'config-change' label to any file changes within the 'docs/config' folder or the docs/README.md file
+config-change:
+- changed-files:
+  - any-glob-to-any-file: ['docs/config/*', 'docs/README.md']
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)